specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2018-03-01-preview/machineLearningServices.json

```diff
@@ -0,0 +1,1462 @@
+          "Create AKS Compute": { "$ref": "./examples/createBasicAKSCompute.json" },
+          "Create a DataFactory Compute": { "$ref": "./examples/createBasicDataFactoryCompute.json" },
+          "Update a BatchAI Compute": { "$ref": "./examples/updateBatchAICompute.json" },
+          "Update a AKS Compute": { "$ref": "./examples/updateAKSCompute.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ComputeNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ComputeDefinitionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Compute creation or update initiated.",
+            "schema": {
+              "$ref": "#/definitions/ComputeResource"
+            }
+          },
+          "201": {
+            "description": "Compute creation or update initiated.",
+            "headers": {
+              "Azure-AsyncOperation": {
+                "description": "URI to poll for asynchronous operation status.",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/ComputeResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the request failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseWrapper"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "OperationalizationClusters",
+          "MachineLearningComputes"
+        ],
+        "operationId": "MachineLearningCompute_Delete",
+        "description": "Deletes specified Machine Learning compute.",
+        "x-ms-examples": {
+          "Delete Compute": { "$ref": "./examples/deleteCompute.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ComputeNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the request was well-formed and received properly."
+          },
+          "202": {
+            "description": "Compute deletion initiated.",
+            "headers": {
+              "Azure-AsyncOperation": {
```

Comment:
If this operation is a polling/async operation - then it should use the [long-running-operation extension](https://github.com/Azure/autorest/tree/master/docs/extensions#x-ms-long-running-operation).

Reply:
Added